src/models/version.rs

```diff
@@ -85,6 +85,8 @@ pub struct NewVersion<'a> {
     crate_id: i32,
     #[builder(start_fn)]
     num: &'a str,
+    #[builder(default = strip_build_metadata(num))]
+    num_no_build: &'a str,
```

Comment:
argh, I screwed this up... this part of the diff should've been in the previous PR instead. every new version is currently inserted with `num_no_build = NULL`, which makes the `set not null` migration in this PR fail 🤦‍♂️ 

```sql
update versions
set num_no_build = split_part(num, '+', 1)
where num_no_build IS NULL;
```

I will run ⬆️ right before deploying, which should be enough for this to deploy cleanly...

Reply:
alright, that worked. all good now! :)